src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+
+  let newState = { ...state };
```

Comment:
Checklist 5. [NAMING] - use proper names for object copy.

Need to emhasize that object is a copy of initial.

Reply:
newState -> currentState

Reply:
Hello Volodymyr, renamed newState to currentState.